### PR TITLE
SONARPY-1943 TypeShed build inside the docker container

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**

### CODE REVIEW

File: python-frontend/typeshed_serializer/docker/cmd.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cd /sonar-python/python-frontend/typeshed_serializer
+# Recreate the env to make sure that latest dependencies will be downloaded
+python -m tox --recreate --notest
+python runners/tox_runner.py --skip_tests=true
```
Comment (ghislainpiot) [inline python-frontend/typeshed_serializer/docker/cmd.sh:6]: Missing execution permissions for this file

File: python-frontend/typeshed_serializer/build-with-docker.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export USER_ID="$(id -u)"
+export GROUP_ID="$(id -g)"
+docker compose -f ./docker/docker-compose.yml --project-directory ./docker up
```
Comment (ghislainpiot) [inline python-frontend/typeshed_serializer/build-with-docker.sh:5]: Does docker compose go "down" automatically after finishing the serialization ?

File: python-frontend/typeshed_serializer/build-with-docker.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export USER_ID="$(id -u)"
+export GROUP_ID="$(id -g)"
```
Comment (ghislainpiot) [inline python-frontend/typeshed_serializer/build-with-docker.sh:4]: Unused code here

**Review by ghislainpiot — APPROVED** [review]
LGTM! I left a few comments

File: python-frontend/typeshed_serializer/build-with-docker.sh (hunk 3 above)
Comment (maksim-grebeniuk-sonarsource) [reply python-frontend/typeshed_serializer/build-with-docker.sh:4]: it is used in docker-compose.yml user section

File: python-frontend/typeshed_serializer/build-with-docker.sh (hunk 2 above)
Comment (maksim-grebeniuk-sonarsource) [reply python-frontend/typeshed_serializer/build-with-docker.sh:5]: yes, the container will stop

**Pull request merged**